src/averageAges.js

```diff
@@ -37,7 +48,19 @@ function calculateMenAverageAge(people, century) {
  * @return {number}
  */
 function calculateWomenAverageAge(people, withChildren) {
-  // write code here
+  const motherAges = people.filter((person) => {
+    return (withChildren
+      ? person.sex === 'f' && people.find(
+        (child) => child.mother === person.name)
+      : person.sex === 'f');
+  });
+
+  const sumOfMotherAges
+  = motherAges.map((person1) => {
```

Comment:
```suggestion
  const motherAges = mothers
    .map((mother) => {
```
Same here. 
Also, why is it person1 if it's the only person?